openforcefield/meta.yaml

```diff
@@ -25,9 +26,19 @@ requirements:
     - networkx >=2.0
     - lxml
     - icu 58*  # This is a lxml dependency but sometimes conda installs version 56
+    - packaging
+    # Should these be optional?
     - rdkit
     - ambertools
     #- parmed # uncomment when new ambermini (without parmed) is built
+    # Should we require this? It's useful for tests and examples, but not strictly required
+    - openmmtools # If we remove this we need to install openmm (which minimum version?)
```

Comment:
Are we using `openmmtools`? Maybe we should just install `openmm` instead?

Reply:
I don't think we use `openmmtools`. 
![image](https://user-images.githubusercontent.com/3656088/50730075-bbd6a280-1112-11e9-8c76-2b41cb4e07ac.png)
Let's just change this to `openmm`.

Reply:
Actually no, I just copied from the recipe in the source repo, although there's a weird reference to `openmmtools.topology` in the `SMIRNOFF_comparison` scripts.

I'll substitute it with `openmm` without minimum version for now.